.travis.yml

```diff
@@ -1,8 +1,9 @@
 
+dist: xenial
 language: python
 python:
-  - 3.5
```

Comment:
Unless there's anything backwards incompatible with 3.5 that we'd be looking to include I'd prefer to keep it (and of course add 3.7).

(For example, on a purely selfish note, for my teaching I'd need to get anaconda reinstalled on all our lab machines if we dropped 3.5.)

Reply:
Ignore this I've just seen https://github.com/Axelrod-Python/Axelrod/pull/1233#issuecomment-453702661

Reply:
I'm fine with keeping 3.5 if you need it. We just have to use the comment form of type hint, which we've already done in some cases.

Reply:
If it's easy enough it could be nicer to keep things a bit more backwards compatible but I'm happy with whatever you prefer.